Fix Identifier value used when configuring Boomi integration.

sjwaight opened the pull request:
Requires content (text) edit and a screenshot (image) change.

azurecla commented:
Hi __@adfteam__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

sjwaight committed:
Add files via upload

Fixed Identifier value to match update to written content.

sjwaight committed:
Update active-directory-saas-boomi-tutorial.md

Fix Identifier value so that setup for Boomi works and avoids a AADSTS70001 error when attempting to log in.
